src/main/play-26/resources/reference.conf

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 HM Revenue & Customs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+play.modules.enabled += "uk.gov.hmrc.cache.MongoCachingModule"
+
+cache {
+  expiryInMinutes = null
```

Comment:
`Configuration.get[Optional[Duration]]` used in provider require to explicit put null in configuration.

It this key is not set configuration provider would use new key `expire`

Reply:
For compatibility reasons. It won't return nested `Option`. In 2.6 line `get[T]` would return exac value `T` and would throw on missing key. Even for `get[Option[T]]` fit this line is not present `2.6` line would throw missing line instead returning `None`. Only possibility is to set explicitly `null` for configuration key and then ask configuration for `Option[T]`. If someone in application.conf would set value up it would take that value over value in `reference.conf`